### DISCUSSION TITLE
Histogram editing?

### DISCUSSION THREAD

**Discussion by zigaturk** [Aries85/LightZone#336] [Ideas]
This program has, IMHO, the most logical and intuitive user interface of them all. But it needs to fix two things:

1) Camera profiles: it should be able to develop the RAW files as per camera profile. A separate tool tab on the right is needed for that.
2) Where is the option to edit the histogram directly? It is a must-have!

Keep up the good work!

**Comment by sfink16** [comment]
> This program has, IMHO, the most logical and intuitive user interface of them all. But it needs to fix two things:
> 
>     1. Camera profiles: it should be able to develop the RAW files as per camera profile. A separate tool tab on the right is needed for that.
> 
>     2. Where is the option to edit the histogram directly? It is a must-have!

@zigaturk The person who used to do Raw Tone Curves (RTCs) is no longer active in this project.  However, there is both written and YouTube videos on how to creat you very own "camera profiles" (RTCs).  Below are the links:
https://github.com/ktgw0316/LightZone/wiki/Creating-Raw-Tone-Curves
https://www.youtube.com/watch?v=i4U8Hv612Gw

Are you referring to what Darktable does to change the exposure of the histogram by moving it?  If not, I'm not sure what you are referring to.  Please elaborate.  Perhaps viewing these videos might help --> https://www.youtube.com/@thelightzoneproject1993/videos
